### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by lianghao2000** [sanghviyashiitb/poisson-deblurring#2]
I wrote the codes for training and synthetic DataSet according to the network's code provided by the author. I use the levin' datasets as validation set. In the training processing, psnr is still equal 2, or small value. For the loss function, 
```python
criterion = nn.L1Loss()
x_rec_list = p4ip_net(y, kernel, peak)
x_rec = x_rec_list[-1]
loss = criterion(shapr, x_rec)
loss.backward()
```

**Comment by sanghviyashiitb** [comment]
I'll upload the original training code I used in a couple of days. Meanwhile, a couple of things you might want to check which might be causing the issue:
1) Are you normalizing "y"? If yes, then don't normalize the input and let the input be y = Poisson (alpha*H*x). The clean image "x" is also normalized to [0,1] instead of [0,255]
2) Make sure the kernel is normalized to one - both during training and test. It is one of the assumptions built into the network. 

If these 2 are not the causing the issue, hang tight and I'll upload the original training code soon.

**Comment by lianghao2000** [comment]
Thanks for your reply

**Comment by sanghviyashiitb** [comment]
Hi,
the training code has been uploaded. Please let me know if there are any issues with it and feel free to ask any more questions.
